corehq/apps/accounting/tests/test_autopay.py

```diff
@@ -93,6 +94,7 @@ def _generate_invoices(cls):
         tasks.calculate_users_in_all_domains(invoice_date)
         tasks.generate_invoices_based_on_date(invoice_date)
 
+    @mocked_stripe_api_decorator
```

Comment:
contextmanager decorators must be called to create a decorator
```suggestion
    @mocked_stripe_api_decorator()
```

Reply:
Thanks Daniel!! `contextmanager decorators must be called to create a decorator` !! I'll try to remember this.